azurerm/resource_arm_servicebus_queue_test.go

```diff
@@ -551,3 +578,26 @@ resource "azurerm_servicebus_queue" "test" {
 }
 `, rInt, location, rInt, rInt)
 }
+
+func testAccAzureRMServiceBusQueue_maxDeliveryCount(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+    name = "acctestRG-%d"
+    location = "%s"
+}
+
+resource "azurerm_servicebus_namespace" "test" {
+    name = "acctestservicebusnamespace-%d"
+    resource_group_name = "${azurerm_resource_group.test.name}"
+    location = "${azurerm_resource_group.test.location}"
+    sku = "standard"
+}
+
+resource "azurerm_servicebus_queue" "test" {
+    name = "acctestservicebusqueue-%d"
+    resource_group_name = "${azurerm_resource_group.test.name}"
+    namespace_name = "${azurerm_servicebus_namespace.test.name}"
+    maxDeliveryCount = 20
```

Comment:
can we update this to `max_delivery_count` to match the schema field?

Reply:
Sure.